services/docker/docker-size.service.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const prettyBytes = require('pretty-bytes')
+const { anyInteger } = require('../validators')
+const { buildDockerUrl, getDockerHubUser } = require('./docker-helpers')
+const { BaseJsonService } = require('..')
+const { NotFound } = require('..')
+
+const buildSchema = Joi.object({
+  name: Joi.string(),
```

Comment:
```suggestion
  name: Joi.string().required(),
```

Reply:
Is this actually the case if the object itself is required?

Reply:
just because there's a `required` on the object doesn't mean that object can't have optional fields, and currently this schema is defining `name` as optional. Our strategy is to have strict validation on these schemas, so if `name` is a value that the service is expecting to exist on this object and will be using, then yes, it is needed.

Reply:
Understood.